test/Microsoft.TestPlatform.AcceptanceTests/ExecutionTests.cs

```diff
@@ -92,15 +93,15 @@ public void RunMultipleTestAssembliesInParallel(string runnerFramework, string t
         }
 
         [CustomDataTestMethod]
-        [NETFullTargetFramework]
+        [NETFullTargetFramework(true, true)]
```

Comment:
Readability: Change it to `[NETFullTargetFramework(inIsolation:true, inProcess:true)]`.

Reply:
Fixed